packages/components/src/buttons/src/EmbeddedButton.tsx

```diff
@@ -26,11 +26,20 @@ import styles from "./EmbeddedButton.module.css";
 export const GlobalEmbeddedButtonCssSelector = "hop-EmbeddedButton";
 
 export interface EmbeddedButtonProps extends StyledComponentProps<RACButtonProps> {
+    /**
+     * Whether the EmbeddedButton should show a selected state.
+     */
+    isSelected?: boolean;
     /**
      * The size of the EmbeddedButton.
      * @default "md"
      */
     size?: ResponsiveProp<"md" | "lg">;
+    /**
+     * The visual style of the EmbeddedButton.
+     * @default "neutral"
+     */
+    variant?: "neutral" | "subdued" | "progress" | "positive" | "caution" | "negative" | "option1" | "option2" | "option3" | "option4" | "option5" | "option6";
```

Comment:
I imagine these variants are for the tag component, so why not pass a class via TagContext and then style the component?

Reply:
I thought it would be cleaner in here and if ever it's needed for the embeddedButton, then it would be available. This will mostly likely only be used internally anyway.